### ISSUE TITLE
"var define" breaks builder logic

### ISSUE THREAD

**Issue by finom** [systemjs/builder#344]
This is about "Uncaught Module X not present." as well but this is another bug. When you define ``define`` variable somewhere, the builder doesn't work correctly.

```js
(function() {
   var f = function() {
      var define = 'this variable is not about AMD stuff';
   }

   define(function() {
      // AMD stuff 
   });
})();
```

**Comment by guybedford** [comment]
Thanks!

**Comment by guybedford** [comment]
Released in 0.14.8.

**Comment by finom** [comment]
Cool.
